packages/ng/dialog/dialog.service.ts

```diff
@@ -50,6 +52,10 @@ export class LuDialogService {
 			...(config.cdkConfigOverride || {}),
 		});
 
+		if (cdkRef.componentRef) {
+			this.#renderer.setStyle(cdkRef.componentRef.location.nativeElement, 'display', 'contents');
+		}
```

Comment:
Can't we get a renderer using `cdkRef.componentRef.injector.get(Renderer2)` ?

Reply:
We can't because we're in a service so we don't have access to `Renderer2`

Reply:
Oh wait, I just realized you're right indeed 😄 I read that wrong and thought you meant to just use inject.